src/projectscene/view/tracksitemsview/labelcontextmenumodel.cpp

```
@@ -22,12 +22,43 @@ void LabelContextMenuModel::load()
         return item;
     };
 
+    MenuItemList cutAndItems {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)